python/dllib/src/bigdl/dllib/nncontext.py

```diff
@@ -275,27 +275,30 @@ def init_spark_standalone(num_executors,
 
 def init_spark_on_k8s(master,
                       container_image,
+                      conda_name,
                       num_executors,
                       executor_cores,
                       executor_memory="2g",
                       driver_memory="1g",
                       driver_cores=4,
                       extra_executor_memory_for_ray=None,
                       extra_python_lib=None,
+                      penv_archive=None,
```

Comment:
Shall we remove this argument for penv_archive for k8s and yarn as well?
As the executor path is hardcoded as python_env, the penv_archive provided by the user wouldn't follow this name and there will be error.

Reply:
It's not conflict. "penv_archive" is a python env .tar path. Either "penv_archive" or conda pack default env .tar will be specified as `--archives env.tar#python_env`.

Reply:
archive = "{}#{}".format(penv_archive, executor_python_env)
But how to guarantee the user specified penv_archieve has `python_env` as we expect?